### PR TITLE
Feature 28 - Exchange functions

### LINKED ISSUE #28: Feature - Exchange functions
## Checklist ✅

- [x] The issue is present in the latest release.
- [x] I have searched the [issues](https://github.com/Platzi-Master-C8/gethired-base/issues) of this repository and believe that this is not a duplicate.

## Summary 💡

We need a function that exchanges the currencies

## Requirements 🌈

- Function that exchanges the currencies

## Motivation 🔦

We need a function that exchanges the currencies

### PR DESCRIPTION
**Pull request by kevfarid**
## Checklist

- [x] I have followed the [contributor guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md)
- [x] Tests are included
- [x] Documentation is changed or added
- [x] Related issue has been created
- [x] Commits messages follows [commit guideline](https://github.com/Platzi-Master-C8/gethired-base/blob/main/CONTRIBUTING.md/#Commits)

### Type of PR

- [x] Feature
- [ ] Bugfix
- [ ] Hotfix
- [ ] Chore

### Merge strategy

- [x] Squash (default)
- [ ] Merge commit (branch merges, usually used for merges to master)
- [ ] Rebase (preserve individual commits without merge commit)

### Relevant tickets

resolves #28

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = {
             App: path.resolve(__dirname, './src/app'),
             Helpers: path.resolve(__dirname, './src/helpers'),
             Services: path.resolve(__dirname, './src/services'),
+            Libs: path.resolve(__dirname, './src/libs')
```
Comment (johannpereze) [inline webpack.config.js:26]: Please add this Alias to the packake.json so it resolves on windows

**Review by johannpereze — CHANGES_REQUESTED** [review]
Please modify package.json to avoid problems on windows

File: webpack.config.js (hunk 1 above)
Comment (kevfarid) [reply webpack.config.js:26]: What for?

**Review by johannpereze — DISMISSED** [review]
LGTM

**Review by johannpereze — APPROVED** [review]

**Pull request merged**